config/settings.yml

```diff
@@ -1204,6 +1204,7 @@
       :starting_timeout: 10.minutes
       :stopping_timeout: 10.minutes
     :embedded_ansible_worker:
+      :starting_timeout: 20.minutes
```

Comment:
Yeah, I don't know.  As long as this timeout is used for the initial setup, a task has variable number of things it does, I'd be concerned with having ansible workers that can never start.  I'd rather be cautious here.